ui/modal/info_modal.go

```diff
@@ -212,8 +223,6 @@ func (in *InfoModal) actionButtonsLayout() layout.Widget {
 					}
 
 					in.btnPositve.Text = in.positiveButtonText
-					in.btnPositve.Background, in.btnPositve.Color = in.Theme.Color.Surface, in.Theme.Color.Primary
```

Comment:
A default background and color should be set so for pages without explicit positive button styles.

![Screenshot 2021-09-14 at 10 21 32](https://user-images.githubusercontent.com/19960200/133231746-15bc6326-6277-4a15-bf1e-9e9807784c74.png)